src/summary/history/patient-notes/patient-notes-card-level-2.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { match } from "react-router";
+import NotesCard from "./notes-card.component";
+import Styles from "./patient-notes-style.css";
+import { getEncounters } from "./encounter.resource";
+
+export default function PatientNotes(props: PatientNotesCardProps) {
+  const properties = {
+    showMore: true,
+    width: "100.0rem",
+    Headers: {
+      date: "DATE",
```

Comment:
It's slightly nicer to use `text-transform: uppercase;` rather than capitalize the words in text. The reason is that it doesn't require the translator to know that it should be uppercase, and also because screen readers read all caps words as acronyms, instead of as words. Which is quite annoying. Would be good to apply some CSS